cpg-language-python/src/test/kotlin/de/fraunhofer/aisec/cpg/frontends/python/PythonFrontendTest.kt

```diff
@@ -1376,6 +1377,93 @@ class PythonFrontendTest : BaseTest() {
         assertEquals(2L, fStmtRhsThird.value)
     }
 
+    @Test
+    fun testFormattedValues() {
+        val topLevel = Path.of("src", "test", "resources", "python")
+        val tu =
+            analyzeAndGetFirstTU(
+                listOf(topLevel.resolve("datatypes.py").toFile()),
+                topLevel,
+                true
+            ) {
+                it.registerLanguage<PythonLanguage>()
+            }
+        assertNotNull(tu)
+        val namespace = tu.namespaces.singleOrNull()
+        assertNotNull(namespace)
+
+        // Test for g = f'Number: {42:.2f}'
+        val gStmt = namespace.statements[6]
+        assertIs<AssignExpression>(gStmt)
+        val gStmtRhs = gStmt.rhs.singleOrNull()
+        assertIs<BinaryOperator>(gStmtRhs)
+        val gFormatCall = gStmtRhs.rhs
+        assertIs<CallExpression>(gFormatCall)
+        assertEquals("format", gFormatCall.name.localName)
+        val gArguments = gFormatCall.arguments
+        assertEquals(2, gArguments.size)
+        assertIs<Literal<*>>(gArguments[0])
+        assertEquals(42.toLong(), gArguments[0].value.value)
```

Comment:
```suggestion
                assertLiteralValue(42.toLong(), gArguments[0])
```
Please also change the other occurrences.